lib/util/constants.js

```diff
@@ -10,40 +10,10 @@ module.exports.WS_SERVER = (environment, region) => {
     ? `wss://${region}.vss.twilio.com/signaling`
     : `wss://${region}.vss.${environment}.twilio.com/signaling`;
 };
-module.exports.ECS_SERVER = (environment, realm) => {
```

Comment:
while at it, see if you can fix the WS_SERVER as well (by using new insights endpoint)

Reply:
@makarandp0 New insights endpoint is not available in all regions yet.